sweep_config.json

```diff
@@ -0,0 +1,29 @@
+{
+    "method": "random",
+    "metric": {
+        "name": "loss",
+        "goal": "minimize"
+    },
+    "parameters": {
+        "optimizer": {
+            "values": [
+                "adam",
+                "sgd"
+            ]
+        },
+        "fc_layer_size": {
+            "values": [
+                128,
+                256,
+                512
+            ]
+        },
+        "dropout": {
+            "values": [
+                0.3,
+                0.4,
+                0.5
```

Comment:
Very high dropout, actually seems like deBERTa v. 2 removed the dropout during pertaining and add it in during fine-tuning.